.github/workflows/ci.yml

```
@@ -0,0 +1,70 @@
+name: CI
+on: push
+jobs:
+  test:
+    name: Test
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        ruby_version:
+          - 2.7
+          - 3.0
+          - 3.1
+          - 3.2
```

The `3.0` value needs to be quoted as a string, otherwise YAML ends up normalizing it to `3`, which the `ruby/setup-ruby` action sees as the latest `3.x.x` version (currently 3.2.2).

For good measure, we might as well quote all of them ^_^

```suggestion
          - "2.7"
          - "3.0"
          - "3.1"
          - "3.2"
```